website/client/components/inventory/equipment/attributesGrid.vue

```diff
@@ -0,0 +1,168 @@
+<template lang="pug">
+  div.attributes-group
+    .popover-content-attr(v-for="attr in ATTRIBUTES", :key="attr")
+      .group-content
+        span.popover-content-attr-cell.key(:class="{'hasValue': hasSumValue(attr) }") {{ `${$t(attr)}: ` }}
+        span.popover-content-attr-cell.label.value(:class="{'green': hasSumValue(attr) }") {{ `${stats.sum[attr]}` }}
+        span.popover-content-attr-cell.label.bold(:class="{'hasValue': hasGearValue(attr) }") Gear:
+        span.popover-content-attr-cell.label(:class="{'hasValue': hasGearValue(attr) }") {{ stats.gear[attr] }}
+        span.popover-content-attr-cell.label.bold(:class="{'hasValue': hasClassBonus(attr) }") Bonus:
```

Comment:
These Gear and Bonus strings should be translatable.

If we use "Class Bonus", the string `classEquipBonus` can be used:
https://github.com/HabitRPG/habitica/blob/028da1d6a9e1d64865dee8ab87289e121efbdd09/website/common/locales/en/character.json#L65.json